src/iface-linux.c

```diff
@@ -110,7 +107,13 @@ static void netlink_callback(CattaNetlink *nl, struct nlmsghdr *n, void* userdat
 
         /* Handle interface attributes */
         l = NLMSG_PAYLOAD(n, sizeof(struct ifinfomsg));
-        a = IFLA_RTA(ifinfomsg);
+        char ifinfomsg_char[sizeof(struct ifinfomsg)];
```

Comment:
Lets talk about this particular change...I am not sure the problem. The code changes look quite complex to simply get rid of a warning...